template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazConfigProvider.java

```diff
@@ -35,11 +35,14 @@
 
     @Override
     public Map<String, ApiClusterTemplateService> getAdditionalServices(CmTemplateProcessor cmTemplateProcessor, TemplatePreparationObject source) {
-        ApiClusterTemplateService coreSettings = createTemplate();
-        Set<HostgroupView> hostgroupViews = source.getHostgroupViews();
-        return hostgroupViews.stream()
-                .filter(hg -> InstanceGroupType.CORE.equals(hg.getInstanceGroupType()))
-                .collect(Collectors.toMap(HostgroupView::getName, v -> coreSettings));
+        if (isConfigurationNeeded(cmTemplateProcessor, source)) {
```

Comment:
It looks like that this was not properly covered with test, please cover it in this fix.

Reply:
Done